src/main/java/com/gmail/nossr50/skills/herbalism/HerbalismManager.java

```diff
@@ -757,69 +757,69 @@ private boolean processGreenThumbPlants(BlockState blockState, BlockBreakEvent b
             return false;
         }
 
-        //If the ageable is NOT mature and the player is NOT using a hoe, abort
-
         Player player = getPlayer();
         PlayerInventory playerInventory = player.getInventory();
         Material seed;
 
         switch (blockState.getType().getKey().getKey().toLowerCase(Locale.ROOT)) {
             case "carrots":
-                seed = Material.matchMaterial("CARROT");
```

Comment:
We use `matchMaterial` to maintain compatibility with older versions. 
Would you mind changing it back?